### DISCUSSION TITLE
i'm try to buy proxies but can't

### DISCUSSION THREAD

**Discussion by wolfslender** [KevinBytesTheDust/CVAmp#224] [Q&A]
I'm trying to buy proxies but the payment method rejects me, does anyone know why?

**Comment by jlplenio** [comment]
Hey you. I have not heard of this problem before. I recommend sending an email to the [webshare](https://www.webshare.io/?referral_code=w6nfvip4qp3g) support. They are rather quick to answer and truly attentive!
